.travis.yml

```diff
@@ -5,13 +5,16 @@ node_js:
 
 services:
   - docker
-
+env:
+  global:
+    - GROUP=deploy.razee.io
+    - VERSION=v1alpha2
 script:
   # Audit npm packages. Fail build whan a PR audit fails, otherwise report the vulnerability and proceed.
   - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then npx audit-ci --low; else npx audit-ci --low || true; fi
   - npm run lint
   - npm test
-  - docker build --rm -t "quay.io/razee/remoteresources3:${TRAVIS_COMMIT}" .
+  - docker build --rm -t "quay.io/razee/remoteresources3:${TRAVIS_COMMIT}" --build-arg GROUP=${GROUP} --build-arg VERSION=${VERSION} .
```

Comment:
is it intentional that the build-arg defined here is `GROUP` and `VERSION` but in the dockerfile is `GROUP_ARG` and `VERSION_ARG`? i thought these had to match.

Reply:
yes you're right

Reply:
im right that they should match, or that it is intentional?

Reply:
ha sorry, yes they should match. That was a last minute change.